extensions/calDotCom/types.ts

```diff
@@ -0,0 +1,8 @@
+export enum TriggerEvent {
+  BOOKING_CREATED = 'BOOKING_CREATED',
+}
+export interface CalComWebhookPayload {
+  triggerEvent: TriggerEvent
+  createdAt: string
+  payload: { uid: string; bookingId: number }
+}
```

Comment:
Does it have value to explicitly define the `TriggerEvent` type here? It's convenient for developers for code IntelliSense but we are missing the other 2 trigger events then.

Reply:
@nckhell Hi, indeed I forgot to include the other 2 types. Do you want me to include the rest or remove it completely?